tower_cli/resources/notification_template.py

```diff
@@ -157,6 +158,11 @@ def _separate(self, kwargs):
             if field in kwargs:
                 result[field] = kwargs.pop(field)
                 if field in Resource.json_fields:
+
+                    # If result[field] is not a string we can continue on
+                    if not isinstance(result[field], str):
+                        continue
```

Comment:
I worry that this might cause more problems... If you are going to check whether something is a string or not, the best collection of string types is `six.string_types`, because the current code will not be python 2 compatible IIRC.

Also, what if it's neither a string or a dict? This code would pass it over silently, but it's probably better to throw an error in those cases.